DoSomethingArchive/legacy-website
# Taxonomy

#### What does this PR do?

Enable the retrieve endpoint for the taxonomy_term resource to be checked off by default. 
#### How should this be manually tested?

Go to http://dev.dosomething.org:8888/admin/structure/services/list/drupalapi/resources and make sure retrieve endpoint under taxonomy_term is checked.  
#### What are the relevant tickets?

Fixes #5342
